sensor_msgs/Image image
sensor_msgs/CameraInfo camera_info
sensor_msgs/Image wide_field
sensor_msgs/CameraInfo wide_camera_info

sensor_msgs/PointCloud2 point_cloud
stereo_msgs/DisparityImage disparity_image

---
# The information for the plane that has been detected
tabletop_object_detector/Table table

# The raw clusters detected in the scan 
sensor_msgs/PointCloud[] clusters

# Whether the detection has succeeded or failed
int32 NO_CLOUD_RECEIVED = 1
int32 NO_TABLE = 2
int32 OTHER_ERROR = 3
int32 SUCCESS = 4
int32 result

---
